pyaedt/maxwell.py

```diff
@@ -1947,8 +1962,8 @@ def edit_external_circuit(self, netlist_file_path, schematic_design_name):
         self.oboundary.EditExternalCircuit(netlist_file_path, sources_array, sources_type_array, [], [])
         return True
 
-    @pyaedt_function_handler()
-    def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
+    @pyaedt_function_handler(setupname="name", setuptype="setup_type")
+    def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         """Create an analysis setup for Maxwell 3D or 2D.
 
         Optional arguments are passed along with ``setuptype`` and ``setupname``.
```

Comment:
```suggestion
        Optional arguments are passed using the ``setup_type`` and ``name``
        parameters.
```
I assume these names have to be edited to match the changes in the "Parameters" section.